### ISSUE TITLE
docs: record a short gif demo showing Minerva working in Telegram and add it to the README

### ISSUE THREAD

**Issue by yurijmikhalevich** [move-fast-and-break-things/minerva#14]

**Comment by harshsennnn** [comment]
Hey can u assign me with this issue

**Comment by harshsennnn** [comment]
is there any specific question that we have to use while recording the demo gif ?

**Comment by yurijmikhalevich** [comment]
@harshsennnn, of course! Assigned. Thank you :)

Any tech- or repo- related question works. We may ask it "How do I set up a Telegram bot?" :-D

**Comment by harshsennnn** [comment]
Hey do we have any discord or slack channel where  i can communicate ?
actually facing some issues

**Comment by yurijmikhalevich** [comment]
@harshsennnn, yes, we do. I have sent you a link in Discord DM. Let me know if you haven't received it.

**Comment by yurijmikhalevich** [comment]
@harshsennnn, I have unassigned you from the issue. Let me know if you still want to work on it.

**Comment by santhosh-7777** [comment]
would love to work on this issue under Hactoberfest 25
Can you please assign this issue to me

**Comment by sassm1t** [comment]
Hey @yurijmikhalevich could you assign this issue to me? I am interested in working on it.

**Comment by yurijmikhalevich** [comment]
@santhosh-7777, thank you for being ready to take on so many different tickets! I have assigned two tickets in the Aibyss project to you. We are trying to assign no more than a few tickets to a person at once. Let's see which tickets we have available once you are done with the Aibyss tickets that you've already taken!

@sassm1t, assigned! Thank you!
